Servers/UI/OJS.Servers.Ui/ClientApp/src/common/tree-types.ts

```diff
@@ -1,8 +1,14 @@
+interface IAllowedStrategyTypes {
+    id: number;
+    name: string;
+}
+
 interface ITreeItemType {
     id: string;
     name: string;
     parentId?: string;
     children?: ITreeItemType[];
+    allowedStrategyTypes: IAllowedStrategyTypes[];
```

Comment:
`ITreeItemType` is an abstract, common, global type used for `tree`s.
**Only** the `tree` with the **categories** will have `allowedStrategyTypes`.

This is incorrect.